### ISSUE TITLE
could you upload your train directory and your trainned module

### ISSUE THREAD

**Issue by liangping** [HotaekHan/SSTDNet#2]
thanks

**Comment by HotaekHan** [comment]
Hello :)

well, sorry. I can not upload my dataset and trained model.
because that data-set is not public.
I will upload just sample image instead of whole dataset.
And I would like to upload pre-trained model for some public dataset like COCO or ICDAR as soon as possible.
please check [Howe to use] section in the README.
And please let me know if you have any problem to use my work :)

**Comment by liangping** [comment]
thanks for helping! 

My problem is that I do not know how to prepare training dataset after I read your README. 

> I will upload just sample image instead of whole dataset.
And I would like to upload pre-trained model for some public dataset like COCO or ICDAR as soon as possible.

It will help us to understand intuitively if you upload some sample images and share your public pre-trained model

**Comment by liangping** [comment]
> (xmin, ymin, xmax, ymax, label)
1273.0 935.0 1407.0 1017.0 v1
911.0 893.0 979.0 953.0 v1
984.0 889.0 1053.0 948.0 v1

How to get this numbers from images and What are images look like?

**Comment by liangping** [comment]
Is there similar with your training dataset
[http://pages.ucsd.edu/~ztu/publication/MSRA-TD500.zip](http://pages.ucsd.edu/~ztu/publication/MSRA-TD500.zip)

if so ,how do you represent angel of text box?

**Comment by HotaekHan** [comment]
sorry for late to upload :(
you can get bounding box using annotation tool like labelimg(https://github.com/tzutalin/labelImg)
please check out that repository. may you can understand what I used for train.

I don't consider angle of bounding box. because I don't handle text detection problem.
I re-implemented SSTDNet for object detection problem. so I just use only 4 parameters for bounding box.

**Comment by HotaekHan** [comment]
I just checked MSRA-TD500 dataset. that provide theta to handle rotation of bounding box.
so you just make to learn the theta also.

**Comment by liangping** [comment]
thanks again!
I am trying use SSTD for text detection to improve performance when CTPN can not detect single characters in my images 

anyway, your code is helpful to me!

**Comment by srwpf** [comment]
@liangping have you figure out how to generate training data now? could you please upload some samples?
